src/components/EventCard.tsx

```diff
@@ -15,14 +16,19 @@ export default function EventCard({
   starttime,
   endtime,
   location,
+  timezone,
 }: EventCardProps) {
   return (
     <Link href={`/view-event?eventId=${eventId}`}>
       <div className="flex h-48 flex-col justify-between rounded-md bg-white p-4 shadow-lg">
         <div>
           <h2 className="text-lg font-bold">{title}</h2>
-          <p>Start Time: {starttime}</p>
```

Comment:
Are we removing this?  As in I thought we didn't need this since we only use start and stop to setup the grids.

Reply:
Removed start and end times from event cards.